README.md

```diff
@@ -1,70 +1,230 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="logo.png" alt="logo" width="400"  height="auto" />
+  <br/>
 
-In the project directory, you can run:
+  <h1><b>SPACE-TRAVELERS-HUB</b></h1>
 
-### `npm start`
+</div>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📗 Table of Contents
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 spacetraveler](#-spacetraveler-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-### `npm test`
+<!-- PROJECT DESCRIPTION -->
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+# 📖 spacetraveler<a name="about-project"></a>
 
-### `npm run build`
+Understand how to use medium-fidelity wireframes to create a UI.
+Follow text written requirements (e.g., user stories).
+Use GitFlow to work with other developers in a team.
+Solve simple git conflicts.
+Write clear, professional, and respectful review comments for other team members.
+Explain "why" a change is requested when giving a code review.
+Perform a code review for a team member.
```

Comment:
- [ ] Kindly change these instructions or Lessons. Basically, in this section, you're expected to describe the main project or brief of the project. In the screenshot below, taken from the project requirement, you will find the actual project descriptions which I believe should be in this section. Kindly work on that 👍🏽

<img width="1079" alt="Screenshot 2023-03-18 at 9 58 45 PM" src="https://user-images.githubusercontent.com/50941074/226142623-2b596e83-b169-4d4a-9939-c68eb1fd0312.png">